### PR TITLE
Fixed with semantic accessible element for nav

### PR DESCRIPTION
**Pull request by divyaprakash-Rx**
span tag made accessible

### CODE REVIEW

**Comment by divyaprakash-Rx** [comment]
@praveenscience @AnjaliSharma1234 I have made span tag accessible. Please review and merge PR.

**Comment by praveenscience** [comment]
Sorry, I should have said this before.

1. Please check the contribution guidelines on indenting, commit messages, etc.
2. Issue isn't linked.
3. The real reason why I have started this issue isn't fixed. That's invalid React code.

**Comment by divyaprakash-Rx** [comment]
@praveenscience can u guide me for the same because I have checked it on local server and now each tag work as links.

**Comment by praveenscience** [comment]
@divyaprakash-Rx Yes, they work, but they aren't right. Let me quote you from the issue:

image

When having `<a>` with `href="#"` itself is invalid, if you remove the `href`, it's more than invalid. 😔 We should be using React Router or `<button>` in this case.  That's what I meant. You cannot have `<a>` tag, unless it's sent to an external link.

Plus, the code isn't formatted / indented correctly. I need to add some eslint rules, but yea, I thought of kick-starting it on Saturday. Anyway, thanks for your contribution. Try to check out how to make accessible links, if you aren't getting it, let me know, I'll guide you. 😊

ps: Check out the React documentation.

**Comment by divyaprakash-Rx** [comment]
Ok

**Comment by praveenscience** [comment]
@divyaprakash-Rx Updated the title in the right way. He he.

Hey, it's worth following the [Contribution Guidelines](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) fully. You did miss out a few steps! No worries if you haven't read it and already started, sorry, my bad! 😔

**Comment by divyaprakash-Rx** [comment]
@praveenscience can we use this <Link> tag.
[https://www.geeksforgeeks.org/reactjs-router/](url)

**Comment by praveenscience** [comment]
Almost yea. But please follow the official documentation bro. 😁 Here we need to have the links dynamic. So come up with an idea. 😇

File: src/components/Header/_NavLink.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ import React from "react";
...
   return (
-    <li className={"nav-item" + (Active ? " active" : "")}>
-      <span className="nav-link" onClick={onClick}>
-        {Name}
-      </span>
+      <li className={"nav-item" + (Active ? " active" : "")}>
+          <span className="nav-link">
+      <a  onClick={onClick}>
+                  {Name}
+              </a></span>
```
Comment (praveenscience) [inline src/components/Header/_NavLink.js]: This one's invalid and also can we follow the proper indenting guidelines, please. 😊

**Pull request closed without merging**